bindings/ruby/lib/rock_gazebo/syskit/robot_definition_extension.rb

```diff
@@ -361,22 +361,11 @@ def normalize_name(name)
 
             # @api private
             #
-            # Define devices for all links and sensors in the model
-            def load_gazebo_robot_model(
+            # Describes recursively all sensors and plugins in the model
+            def load_gazebo_robot_submodel(
```

Comment:
I would have named it in the plural, `load_gazebo_robot_submodels`. Besides that, this looks good to me!